example/share-services/databases/services.nix

```diff
@@ -0,0 +1,27 @@
+{ inputs }:
+{ ... }: {
+  services.postgres."pg1" = {
+    enable = true;
+    listen_addresses = "127.0.0.1";
+    initialDatabases = [
+      {
+        name = "sample";
+        schemas = [ "${inputs.northwind}/northwind.sql" ];
+      }
+    ];
+    # Required due to socket length being limited to 100 chars, see: https://github.com/juspay/services-flake/pull/77
+    socketDir = "$HOME/.services/postgres/pg1";
```

Comment:
Wouldn't this break if running two test builds in parallel?

Also: https://github.com/juspay/services-flake/issues/139#issuecomment-1988436374

Reply:
It will, yes! It should probably depend on a tmp path, rather than a pre-defined one. Gotta think about this one.